sw/airborne/firmwares/rotorcraft/guidance/guidance_h.c

```diff
@@ -426,9 +440,15 @@ void guidance_h_from_nav(bool in_flight)
   }
 
   if (nav.horizontal_mode == NAV_HORIZONTAL_MODE_MANUAL) {
```

Comment:
I'm not so sure the NAV_MODE_MANUAL is still very much in use (or if anyone ever used it...)

Reply:
I don't mind removing it, ask Freek ;) https://github.com/paparazzi/paparazzi/pull/1606

Reply:
Not removing for now as it breaks heli_spin_up